src/lib/select/select.ts

```diff
@@ -388,6 +393,13 @@ export class MatSelect extends _MatSelectMixinBase implements AfterContentInit,
     this._disableOptionCentering = coerceBooleanProperty(value);
   }
 
+  /** Define select panel maximum height. */
+  @Input()
+  get panelMaxHeight(): number { return this._panelMaxHeight; }
+  set panelMaxHeight(value: number) {
```

Comment:
I think we can get away with keeping this as `panelMaxHeight: number`. Getters and setters generate a lot of ES5 code and we're trying to move away from them.